qubes/storage/lvm.py

```diff
@@ -30,18 +30,7 @@
 import qubes.storage
 import qubes.utils
 
-
-def check_lvm_version():
-    #Check if lvm is very very old, like in Travis-CI
-    try:
-        lvm_help = subprocess.check_output(['lvm', 'lvcreate', '--help'],
-            stderr=subprocess.DEVNULL).decode()
-        return '--setactivationskip' not in lvm_help
-    except (subprocess.CalledProcessError, FileNotFoundError):
-        pass
-
-lvm_is_very_old = check_lvm_version()
-
+_sudo, _dd, _lvm = 'sudo', 'dd', 'lvm'
```

Comment:
What's the purpose of those variables?

Reply:
It helps catch typos.  If a variable name is misspelled, it will be caught (either by Pylint or at runtime).  A misspelled command could be much harder to debug.

Additionally, if we ever needed to change commands (say by replacing `sudo` with OpenBSD’s `doas`) the change would only need to be made once.